tools/build_api.py

```diff
@@ -235,7 +235,7 @@ def build_project(src_path, build_path, target, toolchain_name,
 
             add_result_to_report(report, cur_result)
 
-        return res
+        return res, toolchain.map_outputs
```

Comment:
This will also reduce the amount of changes in other scripts, which are currently needed to support the tuple